src/SDKs/Batch/Management/Management.Batch/Microsoft.Azure.Management.Batch.csproj

```diff
@@ -2,16 +2,24 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Batch</PackageId>
-    <Description>Provides management capabilities for Azure Batch service accounts.</Description>
+    <Description>Provides management functions for the Microsoft Azure Batch services.</Description>
     <AssemblyTitle>Microsoft Azure Batch Management Library</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.Batch</AssemblyName>
     <VersionPrefix>5.1.0</VersionPrefix>
-    <PackageTags>Microsoft Azure batch management</PackageTags>
+    <PackageTags>Microsoft Azure Batch Management windowsazureofficial</PackageTags>
     <PackageReleaseNotes>For detailed release notes, see: https://aka.ms/batch-net-management-changelog</PackageReleaseNotes>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
```

Comment:
@xingwu1 can you clarify adding all these properties in the project? Also why the version is different in the same project.